LoopKitUI/Views/FractionalQuantityPicker.swift

```diff
@@ -95,6 +86,18 @@ public struct FractionalQuantityPicker: View {
         self.usageContext = usageContext
     }
 
+    private static func fractionForWhole(
+        _ whole: Double,
```

Comment:
Is `whole` being used? Should it be used?

Reply:
Remnant from a previous implementation, can be removed, thanks!